suite-native/receive/src/types.ts

```diff
@@ -0,0 +1,10 @@
+import { DeviceModelInternal } from '@trezor/connect';
+
+export type PaginationCompatibleDeviceModel = DeviceModelInternal.T2B1 | DeviceModelInternal.T2T1;
```

Comment:
Why are only two models compatible?

Reply:
Firmware does not support address chunking for model T1B1. I think the reason is that its screen is to small, so chunking does not fit in there. 

I have put explanation comment here and also changed the type definition, so if there is will be added new model in the future and it will not be handled in the places where is this style used, it will shout.  45dd281869